### PR TITLE
Create 2015-06-23-simple-contour.html

### PR DESCRIPTION
**Pull request by aneda**

### CODE REVIEW

**Comment by aneda** [comment]
@chriddyp is this good?  Also what should I call this:  https://plot.ly/~PlotBot/58.png

File: _posts/python/contour/2015-06-23-simple-contour.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
...28.3933518, 23.96121884,20.08310249,16.75900277, 13.98891967,11.77285319,10.11080332, 9.00277008, 8.44875346, 8.44875346, 9.00277008,10.11080332,11.77285319,13.98891967, 16.75900277,20.08310249,23.96121884,28.3933518, 33.37950139], [ 36.70360111,31.71745152,27.28531856,23.40720222,20.08310249, 17.31301939,15.09695291,13.43490305,12.32686981,11.77285319, 11.77285319,12.32686981,13.43490305,15.09695291,17.31301939, 20.08310249,23.40720222,27.28531856,31.71745152,36.70360111], [ 40.58171745,35.59556787,31.1634349, 27.28531856,23.96121884, 21.19113573,18.97506925,17.31301939,16.20498615,15.65096953, 15.65096953,16.20498615,17.31301939,18.97506925,21.19113573, 23.96121884,27.28531856,31.1634349, 35.59556787,40.58171745], [ 45.01385042,40.02770083,35.59556787,31.71745152,28.3933518, 25.6232687, 23.40720222,21.74515235,20.63711911,20.08310249, 20.08310249,20.63711911,21.74515235,23.40720222,25.6232687, 28.3933518, 31.71745152,35.59556787,40.02770083,45.01385042], [ 50,45.01385042,40.58171745,36.70360111,33.37950139, 30.60941828,28.3933518, 26.73130194,25.6232687, 25.06925208, 25.06925208,25.6232687, 26.73130194,28.3933518, 30.60941828, 33.37950139,36.70360111,40.58171745,45.01385042,50]],
```
Comment (chriddyp) [inline _posts/python/contour/2015-06-23-simple-contour.html]: maybe simpler data, like a 5x5 matrix, that we can break out 5 human-readable lines?

**Comment by aneda** [comment]
@chriddyp is this good?   Or do you want complete off-center circles?

**Comment by chriddyp** [comment]
yeah, looks great :) :dancer:

**Pull request merged**